internal/store/x509_auth_database_users.go

```diff
@@ -24,21 +24,26 @@ import (
 
 //go:generate mockgen -destination=../mocks/mock_x509_certificate_store.go -package=mocks github.com/mongodb/mongocli/internal/store X509CertificateDescriber,X509CertificateSaver,X509CertificateStore
 
-type X509CertificateDescriber interface {
+type X509CertificateConfDescriber interface {
 	X509Configuration(string) (*atlas.CustomerX509, error)
 }
 
-type X509CertificateSaver interface {
+type X509CertificateConfSaver interface {
 	SaveX509Configuration(string, string) (*atlas.CustomerX509, error)
 }
 
 type X509CertificateDisabler interface {
 	DisableX509Configuration(string) error
 }
 
+type UserCertificateLister interface {
```

Comment:
[nit] to avoid confusion 
```suggestion
type DBUserCertificateLister interface {
```